<!-- New Main Building dataset -->
<launch>

  <node pkg="plane_slam" type="plane_slam_node" name="plane_slam_node" output="screen" >
    <param name="subscriber_queue_size" value="4"/>
    <param name="topic_image_visual" value="/camera/rgb/image_color"/>
    <param name="topic_image_depth" value="/camera/depth/image"/>
    <param name="topic_camera_info" value="/camera/depth/camera_info"/>
    <param name="topic_point_cloud" value=""/>
  </node>

</launch>
